using-valtimo/document/custom-case-columns.md

```diff
@@ -0,0 +1,162 @@
+overview.
+
+| <!-- -->                                                      |
+|---------------------------------------------------------------|
+| ![Default case detail list](img/default-case-detail-list.png) |
+| <center>Default case overview</center>                        |
+
+## Default columns
+
+In the environment file, the following default inline code is found for the case overview columns:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+const defaultDefinitionColumns: Array<DefinitionColumn> = [
+    {
+        propertyName: 'sequence',
+        translationKey: 'referenceNumber',
+        sortable: true,
+    },
+    {
+        propertyName: 'createdBy',
+        translationKey: 'createdBy',
+        sortable: true,
+    },
+    {
+        propertyName: 'createdOn',
+        translationKey: 'createdOn',
+        sortable: true,
+        viewType: 'date',
+        default: true,
+    },
+    {
+        propertyName: 'modifiedOn',
+        translationKey: 'lastModified',
+        sortable: true,
+        viewType: 'date',
+    },
+];
+
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    ...
+};
+  ```
+
+## Custom columns
+
+This above code can be used as a basis for custom columns. First add the line `customDefinitionTables: {}` below the
+line `defaultDefinitionTable: defaultDefinitionColumns`:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    customDefinitionTables: {}
+    ...
+};
+  ```
+
+Inside the brackets custom columns can then be defined:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    customDefinitionTables: {
+        'document-definition-name' : [
+            {
+                propertyName: '$.firstName',
+                translationKey: 'document-definition-name.firstName',
+                sortable: false,
+                viewType: 'string',
+                default: true
+            },
+            {
+                propertyName: 'modifiedOn',
+                translationKey: 'lastModified',
+                sortable: true,
+                viewType: 'date'
+            }
+        ]
+    }
+    ...
+};
+  ```
+
+For each case for which custom columns are to be defined, an array is added with the desired columns. The document
+definition id is used as a key to point to this array, i.e.:  `'document-definition-name': []`.
+
+In this array, for each column an object is added with the following properties:
+
+- `propertyName`: The key from the document definition that is to be used to display column data. Default properties
+like `sequence` can be written plainly. Properties nested inside like `firstName` use a `$.` notation, i.e.: 
```

Comment:
instead of saying '`$.` notation,' i would say that a JSON Path expression is used, and then give an example.